### ISSUE TITLE
Year date facet for UO Vets and Latino Roots

### ISSUE THREAD

**Issue by sseymore** [OregonDigital/oregondigital#968]
The archivist for this collection would like to have the year the class was held be a facet on the set page. Is this feasible? All of the items have dates but the decade facet that's already there is useless for this. 

Also, when sorting the items it looks like it's sorting by the date the item was added to Hydra not the date field. Can that get changed to sorting by the metadata in the date field?

**Comment by wickr** [comment]
Yes, a Year facet is feasible, and it's come up in the past. We need to decide what to do when there's a range though (pick only 1st year, pick several, etc.)  It would be on all items/sets that have Date, where the Year could be parsed.

Sorting by 'Date Ascending' should work as you want. 'Recently Added' is the sort with added to Hydra date.

UO Vets items look to be sorting correctly. Latino Roots items don't seem to all have dates, and some have '06/05/2011' which isn't going to sort correctly and should be corrected.

**Comment by sseymore** [comment]
Cool. These should not have ranges but I can check. The Date Descending is not including the items from 2015. They are in compound objects though so it that the reason? 

I can correct the Latino Roots ones. That was data from CDM and probably didn't get edited.

**Comment by wickr** [comment]
Ok, yeah I didn't think these have ranges but since it would affect all items I would want to have a plan.

For Date Descending, are talking about Latino Heritage or UO Vets?  For UO Vets, I see some that have a Date Created of 2014... but nothing at all for Date. Date is the only field being used for Sort (and Decade facet). The compound objects aren't necessarily a problem but for the Sort to work the parent or child items will have to have the Date set.

**Comment by sseymore** [comment]
Ah, I see. I was talking about UO Vets and it looks like it's split between Date and Created like you said. Ok, I'm going to consolidate those into Date and clean up for both collections and then I'll get back to you about the facet and range issue.Thanks!

**Comment by sseymore** [comment]
Ok, I think I got all the dates corrected for the items I could add dates to so the sorting functionality is working. Since the year facet will impact date ranges in all of the collections, can we discuss it at the next meeting? Or at a check in?
